Assets/Scripts/Game/Automap.cs

```diff
@@ -2002,10 +2002,11 @@ private void CreateDungeonGeometryForAutomap()
             {
                 if (elem.name.Contains("DaggerfallDungeon"))
                 {
+                    DFLocation.DungeonBlock[] blocks = GameManager.Instance.PlayerEnterExit.Dungeon.Summary.LocationData.Dungeon.Blocks;
```

Comment:
I had to base the automap on Summary.LocationData.Dungeon.Blocks instead of location.Dungeon.Blocks, because in the latter all block.WaterLevel are always 0.
Maybe I'm missing something.

Reply:
I think this is fine, even with SmallerDungeons enabled. The new dungeon layout is generated at a lower level and should be correct in this data at runtime.